cmd/dagger/functions.go

```diff
@@ -694,6 +702,207 @@ func handleResponse(returnType *modTypeDef, response any, o, e io.Writer) error
 	return err
 }
 
+func handleChangesetResponse(ctx context.Context, dag *dagger.Client, response any) (rerr error) {
+	var changesetID string
+	switch v := response.(type) {
+	case string:
+		changesetID = v
+	default:
+		return fmt.Errorf("unexpected response type for changeset: %T", v)
+	}
+
+	changeset := dag.LoadChangesetFromID(dagger.ChangesetID(changesetID))
+
+	var summary strings.Builder
+	if err := (func() (rerr error) {
+		ctx, span := Tracer().Start(ctx, "analyzing changes")
+		defer telemetry.End(span, func() error { return rerr })
+
+		patch, err := changeset.AsPatch().Contents(ctx)
+		if err != nil {
+			return fmt.Errorf("get patch contents: %w", err)
+		}
+
+		return summarizePatch(idtui.NewOutput(&summary), patch, -1)
+	})(); err != nil {
+		return err
+	}
+
+	exportDest := outputPath
+	if exportDest == "" {
+		exportDest = "."
+
+		var confirm bool
+		form := idtui.NewForm(
```

Comment:
Should we still prompt even if there are no changes?

<img width="708" height="281" alt="image" src="https://github.com/user-attachments/assets/014d390f-ef2e-4fe8-9254-22a39814ac45" />

This feels a bit weird, since it won't do anything if I hit `y`.